js/deer-utils.js

```diff
@@ -105,6 +115,23 @@ export default {
             return label || noLabel
         }
     },
+    postView(entity, matchOn = ["__rerum.generatedBy", "creator"]){
+        let UTILS = this
```

Comment:
why is this necessary?

Reply:
At some point I was having trouble with the scope of `this`.  Setting it at the top of the function strong-arms the scope to `UTILS` so that the key word `this` isn't conflated in loops and what not.  It may not be necessary any more, but if you plan to undo it for `this` make sure to test it and put breakpoints around the uses of `this` so you can be sure it is what you expect.